src/ray/object_manager/object_store_notification_manager.h

```diff
@@ -20,6 +20,13 @@
 
 namespace ray {
 
+struct RayObjectInfo {
```

Comment:
Can we just use the `ObjectInfo` flatbuffer directly instead of defining this other struct? Maybe `ObjectInfoT`?